Nulecule/answers.conf.sample

```diff
@@ -0,0 +1,8 @@
+[atomic-site]
+image = jberkus/atomic-site
+hostport = 4567
+sourcedir = /home/josh/git/redhat/atomic-site-master/source
+datadir = /home/josh/git/redhat/atomic-site-master/data
+[general]
+namespace = default
+provider = docker
```

Comment:
after reading josh's documentation in the blog post it seems like he wanted this here. Might be better to rather have the user generate the answers.conf and populate his/her own values rather than leave this file around?